src/pipecat/audio/audio_io.py

```diff
@@ -0,0 +1,143 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
not sure if this should be checked in here, but it was the fastest way to get things working.  open to figuring out other ways to get this.